### DISCUSSION TITLE
🧩 FEATURE REQUEST: Complexity level sorting

### DISCUSSION THREAD

**Discussion by GRbit** [dailydotdev/daily#1035] [Feature Requests]
Hi there!

First of all, thank you for the service, it looks good, and as a developer I like the technology you used to build that. Great work!

Despite the fact that I love the idea of the app itself, it's barely usable for me. Nowadays web has tons of articles and authors fighting for the user attention. That make a lot of articles which has something that attracts, but are hollow on the inside. 
I tried to filter out the authors that use 100% clickbait or write useless shit (my personal opinion, no offense), but still there are more and more articles that looks to me like this
`SPOONS AND FORKS!!! WHAT TO CHOOSE???`
And then the author tells us something like "Forks are good, but it's hard to eat soup with a fork".

Anyway, coming to the point. Do you think it'll be a good idea to add not only "votes", but something like "complexity level"? For users to mark an article from Junior to Stuff levels. I would just love a feature like this. I believe there are Junior devs that want to grow and it would be nice for them to have a set of articles that they can fully understand, and it will also attract more senior developers, because so far, the set of articles I see is 95% useless for me.

Thank you very much for your attention!
If you think my idea is a dumb one, or hard to implement (but I believe it's not), I would be really happy if you come up with any other solution to the problem I've described.

**Comment by idoshamun** [comment]
Hi @GRbit,

Appreciate the honest feedback. Low-quality content is something we constantly try to get rid of.
It's a hard problem to fix, but we'll do our best to address that.
To be honest I don't think even juniors need it

**Reply by GRbit** [reply]
Thank you for fast reply @idoshamun !

To be honest, I might be exaggerating a bit. My example with spoons and forks was clearly a "low-quality content". At the same time, if people give their likes, for me it means that they needed it (unless these were likes from bots).

I believe, that unless you are a professional tutor, it's just impossible for a senior developer to understand what will be obvious for a junior and what is not. So, I don't try to judge the content if it doesn't give anything new to myself, rather trying to add a metric to give people like me a chance to recommend some advanced content to other experts.
